.pre-commit-config.yaml

```diff
@@ -48,5 +48,13 @@ repos:
       - id: golangci-lint
         args: [--new-from-patch=/tmp/diff.patch]
 
+    # Rules are in .markdownlint.json file
+    # See https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md for rule descriptions
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.35.0
+    hooks:
+    - id: markdownlint-fix
+      args: [--ignore=CHANGELOG.md, --ignore=design/, --ignore=.github/]
```

Comment:
Why do we want to ignore the changelog and the desing folder? 🤔

Reply:
I ignored changelog because one of the rules complains about the release date format:

```
 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "July 17, 2023"]
```

and I figured we'd want more control over the changelog format. But if we want to lint it, I can just disable that rule. 

I ignored the design folder because there were a bunch of changes that had to be made manually and it didn't seem worth it considering we are replacing design docs with proposals.

Reply:
Removed changelog and design directory from ignores.